src/Substrate.ts

```diff
@@ -41,6 +51,7 @@ export class Substrate {
   baseUrl: NonNullable<Configuration["baseUrl"]>;
   apiVersion: NonNullable<Configuration["apiVersion"]>;
   timeout: NonNullable<Configuration["timeout"]>;
+  secrets: NonNullable<Configuration["secrets"]>;
```

Comment:
```suggestion
  secrets: NonNullable<Configuration["secrets"]>;
```
I think you can just remove this line too since you're not writing the field on `Substrate` (just modifying the `additionalHeaders` field